src/play-item.js

```diff
@@ -41,6 +41,11 @@ const playItem = (player, item, suppressPoster = false) => {
   }
 
   player.poster(suppressPoster ? '' : item.poster || '');
+  player.one('audiopostermodechange', () => {
```

Comment:
Could I just check why an event listener is needed here? Does `player.audioPosterMode()` need a tick or two after playItem is called to contain the updated value?

Reply:
audioPosterMode for a playlist item is turned on only on 'loadedmetadata'. Rn, the poster is suppressed before the player can detect whether or not the playlist item is audio-only.